models/Printer.py

```diff
@@ -0,0 +1,27 @@
+from abc import ABC, abstractmethod
+
+class Printer(ABC):
+    REQUIRED_COLOUR_PER_PAGE = 10
+
+    def __init__(self, model, is_color, is_duplex, paper_tray_capacity, paper_count):
+        self.model = model
+        self.is_color = is_color
+        self.is_duplex = is_duplex
+        self.paper_tray_capacity = paper_tray_capacity
+        self.paper_count = paper_count
+
+    @abstractmethod
+    def print(self, pages):
+        pass
+
+    @abstractmethod
+    def load_paper(self, count):
+        pass
+
+    @abstractmethod
+    def get_remaining_pages_count(self):
+        pass
+
+    def __str__(self):
```

Comment:
Not an issue, but rather a proposal - you could use [dataclass](https://docs.python.org/3/library/dataclasses.html) here to remove some boilerplate code. It's also kind of a Lombok equivalent in Python.